xdg.go

```diff
@@ -98,12 +104,13 @@ func Reload() {
 	RuntimeDir = baseDirs.runtime
 	FontDirs = baseDirs.fonts
 	ApplicationDirs = baseDirs.applications
+	StateHome = baseDirs.stateHome
 
 	// Initialize user directories.
 	initUserDirs(Home)
 }
 
-// DataFile returns a suitable location for the specified data file.
+// DataFile returns a suitable location for the specified data (immutable) file.
```

Comment:
Same here. Let's not change the comment.

Reply:
Fixed